### ISSUE TITLE
Update describe function limits and best practices

### ISSUE THREAD

**Issue by Jextic** [processing/p5.js#9105]
### Increasing access

A new lang parameter will be added to ensure screen readers can programmatically identify the language of the text for the `describe()` and `describeElement()`. This will ensure screen readers (mostly for Windows users) can change to the proper screen reader voice language.

### Most appropriate sub-area of p5.js?

- [x] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build process
- [ ] Unit testing
- [ ] Internationalization
- [ ] Friendly errors
- [ ] Other (specify if possible)

### Feature enhancement details

This pertains to one of the items for issue #9100.

The contributor documentations need to be updated to include these key guidelines:

1. TTS engines cannot reliably pronounce transliterations of languages. This can make it hard for the screen reader users to understand the text for the describe functions. Our current workaround to resolve this is to encourage sketch creators to use their native script language when writing the text for describe functions.

2. macOS VoiceOver ignores the lang attribute completely so Latin-script languages (Spanish, German, French, etc) will be read with the English voice so it's likely that words won't be pronounced correctly.

**Comment by Ayush4958** [comment]
Hi @perminder-17 , 
I would like to work on this issue 
as its a new and very interesting area to me 

thanks ...

**Comment by jonsuguiyama** [comment]
Hi, I'd like to work on this issue if it's still available. I'll open a PR soon.

**Comment by perminder-17** [comment]
Assigned, thanks for taking this up @Ayush4958

**Comment by Ayush4958** [comment]
> Assigned, thanks for taking this up [@Ayush4958](https://github.com/Ayush4958)

thanks, I will work on this and 
will raise a PR for it
